### PR TITLE
Darya & Rebecca ChatBot 

### PR DESCRIPTION
**Pull request by DALA746**

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -1,47 +1,120 @@
...
+            </div>
+          </section>
+        `
+      chat.scrollTop = chat.scrollHeight
+    }
+ }
+
+//5.
+    const showFoodOptions = () => {
+      currentQuestion++ // current question is 2
```
Comment (Rosanna86) [inline code/script.js:61]: Good with small comments at each interaction with the bot. Makes it easy for me as a code reviewer to read the code.

File: code/script.js [hunk 2]
```diff
@@ -1,47 +1,120 @@
...
+      setTimeout(() => showMessage(`What sushi would you like to order? `, 'bot'), 1000)
+      inputWrapper.innerHTML = `
+        <button id="small-sushi">8 pieces</button>
+        <button id="big-sushi">12 pieces</button>
+      `
+
+      const smallSushi = document.getElementById('small-sushi')
```
Comment (Rosanna86) [inline code/script.js:68]: Good naming on the variables. Small and big sushi.

File: code/script.js [hunk 3]
```diff
@@ -1,47 +1,120 @@
...
+      currentQuestion++// current question is 3
+      inputWrapper.innerHTML = `
+        <button id="yes">Yes</button>
+        <button id="no">No</button>
+      ` 
+      const yesAnswer = document.getElementById('yes'); 
+      yesAnswer.addEventListener('click', () => nextQuestion('yes'));
+
+      const noAnswer = document.getElementById('no'); 
+      noAnswer.addEventListener('click', () => nextQuestion('no'));
```
Comment (Rosanna86) [inline code/script.js:86]: The function when I click "NO" and chat restarts is good. If you had more time maybe the bot could have answered "Okay I understand, see you next time" and then restart?

File: code/script.js [hunk 4]
```diff
@@ -1,47 +1,120 @@
...
+    
+    }
+
+//3. The bot shows a response to the user 
+    const handleNameInput = () => {
+      const name = nameInput.value
+      showMessage(name, 'user')
+      setTimeout(() => showMessage(`Nice to meet you ${name}`, 'bot'), 1000)
```
Comment (Rosanna86) [inline code/script.js:94]: Nice with the template literals. Makes it more personal.

File: code/script.js [hunk 5]
```diff
@@ -1,47 +1,120 @@
...
+    const greeting = () => {
+      showMessage(`Hi, my name is SushiBot! What's your name?`, 'bot'); // question nr 1
+    }
+    setTimeout(greeting, 1000)
+
+// 9.
+    const ending = (message) => {
+      if (message === 'yes') {
+        showMessage('Thank you for your order', 'bot')
+        setTimeout(() => location.reload(), 5000)
```
Comment (Rosanna86) [inline code/script.js:108]: Good setTime out before reload page. 5000 is not to long.

File: code/style.css [hunk 6]
```diff
@@ -5,18 +5,102 @@
 body {
   margin: 0;
   padding: 0;
-  font-family: 'Montserrat', sans-serif;
-  background: #0026ff;
+  /* background-image: url(./assets/background.jpeg); */
+  background-color: #585858;
+  font-family: 'Palette Mosaic', cursive;
```
Comment (Rosanna86) [inline code/style.css:10]: Perfect font for a sushi place! And the colors as well!

File: code/style.css [hunk 7]
```diff
@@ -5,18 +5,102 @@
...
+ul li{
+  list-style:none;
+  font-family: 'Palette Mosaic', cursive;
+  font-size:5em;
+  letter-spacing:15px;
+  text-shadow:none;
+  animation:animate 1s infinite linear;
+}
+
+@keyframes animate{
```
Comment (Rosanna86) [inline code/style.css:43]: This is really well done with the H1! Had no idea that you could do it like this.

File: code/style.css [hunk 8]
```diff
@@ -37,8 +121,8 @@ input {
   box-sizing: border-box;
   border: none;
   border-radius: 4px 0 0 4px;
-  background: #e5e9ff;
-  color: #0026ff;
+  background: #747474;
```
Comment (Rosanna86) [inline code/style.css:124]: Nice adding background color on input field. Maybe if you had the time - the bubbles also could have that grey color?

File: code/style.css [hunk 9]
```diff
@@ -5,18 +5,102 @@
...
+ul{
+  display:flex;
+  justify-content: center;
+}
+
+
+ul li{
+  list-style:none;
+  font-family: 'Palette Mosaic', cursive;
+  font-size:5em;
```
Comment (Rosanna86) [inline code/style.css:37]: Here you have font-size in unit em and on other places px. Maybe decide on one?

File: code/script.js [hunk 10]
```diff
@@ -1,47 +1,120 @@
...
+              <p>${message}</p>
+            </div>
+            <img src="./assets/user.png" alt="User" />  
+          </section>
+        `
+
+      } else if (sender === 'bot') {
+        chat.innerHTML += ` 
+          <section class="bot-msg">
+          <img src="./assets/sushi-roll-icon.jpeg" alt="">
```
Comment (Rosanna86) [inline code/script.js:49]: This one made my day. Best sushi icon ever!

**Review by Rosanna86 — COMMENTED** [review]
**Overall** 
Great design, really looks like a sushi ordering place. I like the comments in the code, it makes it easier for me as a code reviewer to go through the code.

I understand that time was an issue, but I think you did a great job and got a good interaction between bot and user. 

The code was easy to read and everything was well written and specified with good names. 

🔵  Blue Level (Minimum Requirements) [yes]

🔴  Red Level (Intermediary Goals) [CSS added]

File: code/script.js (hunk 3 above)
Comment (DALA746) [reply code/script.js:86]: Yeah, that's a great idea! When I have time I am gonna add it :)

File: code/style.css (hunk 6 above)
Comment (DALA746) [reply code/style.css:10]: We were trying to find a font that is more Japanese, but couldn't find it. Some really nice fonts costs so we used that one int the project. Do you have some ideas for fonts? :)

File: code/style.css (hunk 7 above)
Comment (DALA746) [reply code/style.css:43]: Found on google, there are a lot of great animation you can do with css!

File: code/style.css (hunk 6 above)
Comment (Rosanna86) [reply code/style.css:10]: This site is good - it has true type fonts. I think you can add it?

https://www.dafont.com/search.php?q=japanese

File: code/style.css (hunk 8 above)
Comment (DALA746) [reply code/style.css:124]: Thanks for designing tips! If you have more tips please shoot me :)

File: code/script.js (hunk 10 above)
Comment (DALA746) [reply code/script.js:49]: Haha :)

File: code/script.js (hunk 3 above)
Comment (DALA746) [reply code/script.js:86]: Yes, great idea!

File: code/style.css (hunk 6 above)
Comment (DALA746) [reply code/style.css:10]: Thanks for tip! :)

**Comment by DALA746** [comment]
Thanks for your feedback! :)

**Pull request closed without merging**